one/src/daemon.rs

```diff
@@ -231,6 +231,53 @@ pub struct DaemonOpts {
         requires = "experimental_features"
     )]
     anchor_poll_retry_count: u64,
+
+    /// Ethereum RPC URLs used for time events validation. Required when connecting to mainnet and uses fallback URLs if not specified for other networks.
+    #[arg(
+        long,
+        use_value_delimiter = true,
+        value_delimiter = ',',
+        env = "CERAMIC_ONE_ETHEREUM_RPC_URLS"
+    )]
+    ethereum_rpc_urls: Vec<String>,
+}
+
+async fn get_rpc_providers(
+    ethereum_rpc_urls: Vec<String>,
+    network: &Network,
+) -> Result<Vec<EthRpcProvider>> {
+    let ethereum_rpc_urls = if ethereum_rpc_urls.is_empty() {
+        network.default_rpc_urls()?
+    } else {
+        ethereum_rpc_urls
+    };
+
+    let mut providers = Vec::new();
+    for url in ethereum_rpc_urls {
+        match HttpEthRpc::try_new(&url).await {
+            Ok(provider) => {
+                let provider: EthRpcProvider = Arc::new(provider);
+                let provider_chain = provider.chain_id();
+                if network
+                    .supported_chain_ids()
+                    .map_or(true, |ids| ids.contains(provider_chain))
+                {
+                    providers.push(provider);
+                } else {
+                    warn!("Eth RPC provider {} uses chainid {} which isn't supported by Ceramic network {:?}", url, provider_chain,network);
+                }
+            }
+            Err(err) => {
+                warn!("failed to create RCP client with url: '{url}': {err}");
```

Comment:
nit : RPC *

Reply:
Fixed in [543](https://github.com/ceramicnetwork/rust-ceramic/pull/543)